### PR TITLE
Add a file for Portland, ND

### PR DESCRIPTION
**Pull request by michaelchristianson-securian**
The purpose of this PR is to add a hometown file for Portland, ND

Some :question: that I want you all to help me answer:
1. If you've been to North Dakota, what did you do? What did you think of it?
1. Do you or your kids play hockey?

### CODE REVIEW

**Comment by kristengemmill** [comment]
1. I believe I was in North Dakota as a child but I don't remember where
2. I do not have kids and I don't play hockey :frowning:

File: michaelchristianson-securian-portland.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Portland, ND
+
+### Restaurants
+* Paula's Cafe
+* Pizza Shop
+
+### Places the locals like
+* Hockey rink
+* Stub's
+* Floyd's
```
Comment (kristengemmill) [inline michaelchristianson-securian-portland.md:10]: @michaelchristianson-securian 
What type of place is Floyd's?

Comment (michaelchristianson-securian) [reply michaelchristianson-securian-portland.md:10]: It's a small town bar -- so is Stub's.

**Comment by michaelchristianson-securian** [comment]
This Pull Request resolves #10

**Review by kristengemmill — APPROVED** [review]
Looks ready to :ship: !

**Pull request merged**